src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.html

```diff
@@ -355,6 +355,21 @@ <h4 *ngIf="isImport" id="jhi-programming-exercise-heading-import" jhiTranslate="
                         </label>
                     </div>
                 </div>
+                <div class="form-group-narrow">
+                    <div class="form-check">
+                        <label class="form-check-label" for="field_showTestNamesToStudents">
```

Comment:
can we have some explanation with a tooltip?

Reply:
yes, done